src/odemis/gui/comp/stream.py

```diff
@@ -1319,6 +1319,7 @@ class StreamBar(wx.Panel):
     def __init__(self, *args, **kwargs):
 
         add_btn = kwargs.pop('add_button', False)
+        add_stream_label = kwargs.pop('label', "ADD STREAM")
```

Comment:
You can drop these changes now that there is a separate button for the overview.